docs/tracing.md

```diff
@@ -26,11 +26,12 @@ identifying elongated function calls.
 
 Firecracker implements instrumentation based tracing via
 [`log`](https://github.com/rust-lang/log) and
-[`log_instrument`](https://github.com/JonathanWoollett-Light/log-instrument),
-outputting a `Trace` level log when entering and exiting every function.
+[`log_instrument`](../src/log-instrument), outputting a `Trace` level log when
+entering and exiting every function.
 
-It is disabled by default at compile-time. Tracing functionality has no impact
-on the release binary.
+Adding traces impacts Firecracker binary size and its performance, so tracing is
+disabled by default. Tracing functionality is also disabled on the available
+release binaries.
```

Comment:
Related to https://github.com/firecracker-microvm/firecracker/pull/4423#discussion_r1477824526 I would suggest:

```suggestion
Tracing is disabled by default as  instrumentation impacts Firecracker's binary size.
Tracing functionality is disabled on the release binaries.
```

Reply:
I modified (almost) as you suggested, thanks.